ts/indicators/wma.src.ts

```diff
@@ -189,8 +175,8 @@ seriesType<Highcharts.WmaIndicator>(
 
             // Switch index for OHLC / Candlestick
             if (isArray(yVal[0])) {
-                index = (params.index);
-                yValue = (yVal[0] as any)[index];
+                index = (params.index as any);
```

Comment:
Maybe `params.index` needs to be non-optional defined.

Reply:
Changed to be non-optional.